src/components/ChatComponents/SuggestedPrompts.tsx

```diff
@@ -0,0 +1,139 @@
+import { ChainType } from "@/chainFactory";
+import { VAULT_VECTOR_STORE_STRATEGY } from "@/constants";
+import React, { useMemo } from "react";
+
+interface NotePrompt {
+  title: string;
+  prompts: string[];
+}
+const SUGGESTED_PROMPTS: Record<string, NotePrompt> = {
+  activeNote: {
+    title: "Active Note Insights",
+    prompts: [
+      `Provide three follow-up questions worded as if I'm asking you based on {activeNote}?`,
+      `What key questions does {activeNote} answer?`,
+      `Give me a quick recap of {{activeNote}} in two sentences.`,
+    ],
+  },
+  quoteNote: {
+    title: "Note Link Chat",
+    prompts: [
+      `Based on [[noteName]], what improvements should we focus on next?`,
```

Comment:
Should we use `<>` to indicate that the user must replace it `[[<your note>]]`?

Reply:
updated.

I originally used this format but removed `<>` because it makes double click to select text to replace more difficult because `<>` is not selected with double clicks. However, I don't think this matters any more because select to replace is tedious to use even with double clicks. We need to follow up to make replace placeholder easier if people use this features a lot.